process/slack_new_member_welcome.md

```diff
@@ -0,0 +1,29 @@
+<H1><img align="left" height=100 src="images/Girlgeek_w_workmark.png"><br> 
+  <em>Hello!</em></H1>
+
+&nbsp;  
+
+
+You're welcome to the Slack Platform of theGlobal PyLadies Community. Kindly do well to introduce yourself, the chapter you belong to and tell us about your involvement with the Python Programming Language.  
+
+
+
+Whiles you can make posts on this platform, you are cautioned to put them in the right channel. The current available channels are:  
```

Comment:
*Edit:*
Reword to -
`While you can make posts on this platform, you are cautioned to put them in the right channel. We have several available channels, such as those listed below. However it may help you engage more effectively by finding the appropriate channel to participate in! Some example channels include: `